torch/csrc/autograd/function.h

```diff
@@ -91,7 +91,7 @@ struct TORCH_API Node : std::enable_shared_from_this<Node> {
   /// a (currently THE) hint to prioritization in the backward() pass, with
   /// higher sequence numbers prioritized before lower sequence numbers.
   explicit Node(
-      uint64_t sequence_nr,
+      int64_t sequence_nr,
```

Comment:
why did you need int64_t here? Normally autograd should not get negative seq_nrs?

Reply:
it's not necessary here, will upd